en/docs/design/api-policies/samples/create-custom-cc-policy.md

```diff
@@ -0,0 +1,215 @@
+            <li>queryParamValue</li>
+        <ul></td>
+    <td>Request</td>
+    <td>Add a query param in the request flow.</td>
+ </tr>
+ <tr>
+    <td>REWRITE_RESOURCE_METHOD</td>
+    <td>
+        <ul>
+            <li>currentMethod</li>
+            <li>updatedMethod</li>
+        </ul></td>
+    <td>Request</td>
+    <td>Change the HTTP method of a resource in the request flow.</td>
+ </tr>
+ <tr>
+    <td>REWRITE_RESOURCE_PATH</td>
+    <td>
+        <ul>
+            <li>resourcePath</li>
+            <li>includeQueryParams</li>
+        </ul></td>
+    <td>Request</td>
+    <td>Rewrite the resource path in the request flow.</td>
+ </tr>
+ <tr>
+    <td>CALL_INTERCEPTOR_SERVICE</td>
+    <td>
+        <ul>
+            <li>interceptorServiceURL</li>
+            <li>includes (this require a comma separated string, which describes what should be included in the request body of the interceptor service)</li>
+        </ul></td>
+    <td>Request Response</td>
+    <td>Call an interceptor service in request or response flow. For more information, visit <a href="{{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/message-transformation/message-transformation-overview">Message Mediation</a>.</td>
+ </tr>
+ <tr>
+    <td>OPA</td>
+    <td>
+        <ul>
+            <li>requestGenerator</li>
+            <li>serverURL</li>
+            <li>accessKey</li>
+            <li>policy</li>
+            <li>rule</li>
+            <li>sendAccessToken</li>
+            <li>additionalProperties</li>
+            <li>maxOpenConnections</li>
+            <li>maxPerRoute</li>
+            <li>connectionTimeout</li>
+        </ul></td>
+    <td>Request</td>
+    <td>Validate the request against Open Policy Agent server.</td>
+ </tr>
+</table>
+
+Custom Choreo Connect policies can use these actions and template or specific value in parameters.
+
+!!! note
+	Choreo Connect policy definitions are in the .gotmpl file extension. Its content type is YAML, templated with [go templates](https://pkg.go.dev/text/template).
+
+## Sample 1 - Custom Call Interceptor Policy
+
+Lets create a new policy using the action CALL_INTERCEPTOR_SERVICE. Following is the content of the default call interceptor policy definition which ships with the distribution. You can download the default call interceptor service policy from the publisher portal.
+
+```yaml tab='ccAddHeader.gotmpl'
+definition:
+ action: CALL_INTERCEPTOR_SERVICE
+ parameters:
+   interceptorServiceURL: {{'{{ .interceptorServiceURL }}'}}
+   includes: {{'{{ .includes }}'}}
+```
+
+Lets say you have an interceptor service that converts XML payload to JSON with the server URL https://xml-to-json-interceptor:8443 and lets create a policy named “XML to JSON Call Interceptor”. Learn more about Choreo Connect interceptors on Message Transformation. Since we only need request and response payload, we can specify includes as “request_body,response_body”. Following the the definition.
```

Comment:
```suggestion
Let's say you have an interceptor service that converts XML payload to JSON with the server URL `https://xml-to-json-interceptor:8443` and let's create a policy named `XML to JSON Call Interceptor`. Learn more about Choreo Connect interceptors on Message Transformation. Since we only need request and response payload, we can specify includes as `request_body,response_body`. Following the definition.
```